zhmcclient/_partition.py

```diff
@@ -773,3 +773,110 @@ def wait_for_status(self, status, status_timeout=None):
                     actual_status, statuses, status_timeout)
 
             time.sleep(1)  # Avoid hot spin loop
+
+    @logged_api_call
+    def increase_crypto_config(self, crypto_adapters,
+                               crypto_domain_configurations):
+        """
+        Adds the specified adapters and/or domain configurations
+        to the crypto configuration of the corresponding partition.
+
+        Authorization requirements:
+
+        * Object-access permission to this Partition.
+        * Task permission to the "Partition Details" task.
+
+        Parameters:
+
+          crypto_adapters (list): Crypto adapters that should be added
+            to the crypto configuration of this partition.
+
+          crypto_domain_configurations (list):
+            List of crypto-domain-configuration objects
+            that should be added to the crypto configuration
+            of this partition. See properties of crypto-domain-configuration
+            objects in section 'Data model' in section 'Partition object'
+            in the :term:`HMC API` book.
```

Comment:
A crypto-domain-configuration object is a tuple of domain index and access-mode. This is simple enough to describe it here.

Specifically, we should mention that the domain index is per adapter, and how to find out unused domain index numbers.